src/functions/assembleSupSub.js

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
@kevinbarabash  We don't have a `utils` folder. If we did, then the `utils.js` file would belong there and I would prefer to avoid updating all the paths that point to `utils.js`. As an alternate, I propose to put this function back into `op.js` but expose it so that it can be called from `operatorname.js`.

Would that be okay?

Reply:
Never mind. It's done.